pkg/apis/authentication/types.go

```diff
@@ -162,3 +162,24 @@ type BoundObjectReference struct {
 	// UID of the referent.
 	UID types.UID
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// SelfSubjectAttributesReview contains the user information that the kube-apiserver has about
+// the user who originated the request.
```

Comment:
tweak this to be more accurate for requests that use impersonation (and copy and updates to the versioned type godoc):

```suggestion
// the user making this request.
```

@deads2k, should this explicitly describe what is returned for requests using impersonation?

Reply:
> @deads2k, should this explicitly describe what is returned for requests using impersonation?

Wouldn't hurt.

Reply:
Fixed.